src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -94,18 +335,98 @@ export class HuiCardPicker extends LitElement {
     });
   }
 
-  private async _cardPicked(ev: Event): Promise<void> {
-    const type = (ev.currentTarget! as CardPickTarget).type;
+  private _cardPicked(ev: Event): void {
+    const config: LovelaceCardConfig = (ev.currentTarget! as CardPickTarget)
+      .config;
 
-    const elClass = await getCardElementClass(type);
```

Comment:
No need for this since I am getting the stub config and setting it on the element